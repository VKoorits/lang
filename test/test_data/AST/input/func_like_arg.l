pow1(pow2(pow3(2,2),2),pow(2,2))
