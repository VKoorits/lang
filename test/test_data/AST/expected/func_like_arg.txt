CALL_0 NULL
CALL_0 :LIST_1 pow1 NULL
LIST_1 :CALL_2 CALL_3 NULL
CALL_2 :LIST_4 pow2 NULL
LIST_4 :CALL_5 ELEM_6 NULL
CALL_5 :LIST_7 pow3 NULL
LIST_7 :ELEM_8 ELEM_9 NULL
ELEM_8 :2 NULL
ELEM_9 :2 NULL
ELEM_6 :2 NULL
CALL_3 :LIST_4 pow NULL
LIST_4 :ELEM_5 ELEM_6 NULL
ELEM_5 :2 NULL
ELEM_6 :2 NULL
